src/aiortc/contrib/media.py

```diff
@@ -161,30 +210,36 @@ def __init__(self, player, kind):
         self._queue = asyncio.Queue()
         self._start = None
 
-    async def recv(self):
+    async def recv(self) -> Union[Frame, Packet]:
         if self.readyState != "live":
             raise MediaStreamError
 
         self._player._start(self)
-        frame = await self._queue.get()
-        if frame is None:
+        data = await self._queue.get()
+        if data is None:
             self.stop()
             raise MediaStreamError
-        frame_time = frame.time
+        if isinstance(data, Frame):
+            data_time = data.time
+        elif isinstance(data, Packet):
+            data_time = data.pts * (
```

Comment:
I wonder if we couldn't just do `float(data.pts * data.time_base)`

Reply:
according `pyav`doc: `time_base` is a [fractions.Fraction](https://docs.python.org/3/library/fractions.html#fractions.Fraction).

I tested and it is possbile :D